megamek/src/megamek/server/totalwarfare/TWGameManager.java

```diff
@@ -8282,6 +8282,19 @@ boolean doSkillCheckManeuver(Entity entity, PilotingRollData roll) {
      * @return true if check succeeds, false otherwise.
      */
     boolean doSkillCheckInPlace(Entity entity, PilotingRollData roll) {
+        return doSkillCheckInPlace(entity, roll, mainPhaseReport);
+    }
+
+    /**
+     * Do a piloting skill check while standing still (during the movement
+     * phase).
+     *
+     * @param entity The <code>Entity</code> that should make the PSR
+     * @param roll   The <code>PilotingRollData</code> to be used for this PSR.
+     * @param vPhaseReport the report vector to write into (sometimes an interim collection)
+     * @return true if check succeeds, false otherwise.
+     */
+    boolean doSkillCheckInPlace(Entity entity, PilotingRollData roll, Vector<Report> vPhaseReport) {
```

Comment:
maybe call the variable vPhaseReport - skillReport so that it is clear this in a internal version of the report

Reply:
We will probably want to do the same for every function that modifies or returns a Report Vector... later.